package/MDAnalysis/analysis/pca.py

```diff
@@ -414,6 +414,42 @@ def transform(self, atomgroup, n_components=None, start=None, stop=None,
 
         return dot
 
+    def project_single_frame(self, components=None):
+        """Runs an inverse-PCA and generates
+           a trajectory that is projected on selected PCs
+
+        Run PCA class before using this function.
+        To project the trajectory onto the first principal component,
+            pca = PCA(universe, select='backbone').run()
+            func = pca.project_single_frame(components=0)
+
+        To apply the projection on-the-fly, may add as a transformation
+            u.trajectory.add_transformations(func)
+
+        Parameters
+        ----------
+        components : int, array, optional
+             Components to be projected onto.
+             The default ``None`` maps onto all components.
+
+        Returns
+        -------
+        Returns the projecting function which transforms a timestep
+        """
+        if components is None:
+            components = range(self.results.p_components.shape[1])
+
+        def wrapped(ts):
+            xyz = self._atoms.positions.ravel() - self._xmean
+            dot = np.dot(xyz, self._p_components[:, components])
```

Comment:
use a more expressive variable name than `dot` (which could also be confused with the numpy function)